src/commons/eventsourcing/cmd.rs

```diff
@@ -20,6 +19,95 @@ use crate::commons::{
 /// CommandSummary for use in history.
 pub trait WithStorableDetails: Storable + Send + Sync {
     fn summary(&self) -> CommandSummary;
+
+    /// Create an instance representing the initialisation command/event.
+    ///
+    /// Should not contain any data, so that we can keep this signature
```

Comment:
Which signature? The function signature? Why is it important to keep it "simple" (and assuming we even agree on the meaning of "simple" in this context...)

Reply:
fixed